src/Arcus.WebApi.Security.Authentication/SharedAccessKey/SharedAccessKeyAuthenticationAttribute.cs

```diff
@@ -8,29 +8,30 @@ namespace Arcus.WebApi.Security.Authentication.SharedAccessKey
     /// <summary>
     /// Authentication filter to secure HTTP requests with shared access keys.
     /// </summary>
-    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Method)] 
+    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Method)]
     public class SharedAccessKeyAuthenticationAttribute : TypeFilterAttribute
     {
         /// <summary>
         /// Initializes a new instance of the <see cref="SharedAccessKeyAuthenticationAttribute"/> class.
         /// </summary>
         /// <param name="secretName">The name of the request header which value must match the stored secret with the same name as the header.</param>
         /// <exception cref="ArgumentException">When the <paramref name="secretName"/> is <c>null</c> or blank.</exception>
-        public SharedAccessKeyAuthenticationAttribute(string secretName) : this(secretName, secretName) { }
+        public SharedAccessKeyAuthenticationAttribute(string secretName) : this(secretName, secretName, secretName) { }
 
         /// <summary>
         /// Initializes a new instance of the <see cref="SharedAccessKeyAuthenticationAttribute"/> class.
         /// </summary>
         /// <param name="headerName">The name of the request header which value must match the stored secret.</param>
+        /// <param name="queryParameterName"></param>
```

Comment:
Can you provide some documentation here please?

Reply:
Will do